### PR TITLE
Fixes #911. Adds 'new' as to create a new project

### LINKED ISSUE #911: Command `donejs new` for creating new projects
Create a new DoneJS CLI command `donejs new` for **creating new things**. The `donejs add` command should be kept and used for additive features to existing projects.

Currently, these commands are "creating": `donejs add app|plugin|generator`.

### PR DESCRIPTION
**Pull request by mjstahl**
@andrejewski Is right and I misread the issue.

`new` has been updated to be an alias for `add`.  Commander's `.alias` function was not used so that they appear to be the different commands with the same functionality.

### CODE REVIEW

**Comment by andrejewski** [comment]
Sorry, I did not get the notification for review and stumbled on to this by chance. I'd like to see some tests for this, but since there are no direct bin/donejs CLI tests this is fine. I'll pull this down locally and try it out before I approve but then I will be content.

File: lib/cli/index.js [hunk 1]
```diff
@@ -5,9 +5,17 @@ var runBinary = require('./run-binary');
 // commands
 var add = require('./cmd-add');
 var help = require('./cmd-help');
+var init = require('./cmd-init');
 
 program.version(require('../../package.json').version);
 
+// donejs new
+program.command('new <folder>')
```
Comment (andrejewski) [inline lib/cli/index.js]: For the `new` command, it should be able to create apps, plugins, or generators; not only new apps. It's essentially the same as add. The word **new** just better articulates the creation effect of commands, which is a convention other CLI tools align with.

File: lib/cli/index.js [hunk 2]
```diff
@@ -5,9 +5,17 @@ var runBinary = require('./run-binary');
...
+  .description('Create and initialize DoneJS application in folder')
+  .action(function(folder, params) {
```
Comment (andrejewski) [inline lib/cli/index.js]: For the `new` command we should also have the usage information like `add` does.

**Review by andrejewski — CHANGES_REQUESTED** [review]
`new` should be an alias of `add` and have its own usage docstring

**Review by andrejewski — APPROVED** [review]

**Review by chasenlehara — CHANGES_REQUESTED** [review]
Code looks good, just needs docs and [tests](https://github.com/donejs/donejs/tree/master/test)! I think they should be added to this section: https://github.com/donejs/donejs/blob/master/docs/apis.md#cli-and-generators

**Comment by frank-dspeed** [comment]
As pointed out in the related issue 
- #911. Adds 'new' as to create a new project 
i would suggest to talk about #995 and don't change existing docs and API
> as also said in the issue that plugin and generate always create new projects that's not true some generators are also modifying existing projects i think this will have unexpected side effects

**Pull request closed without merging**